libs/dao/src/main/java/com/akto/dto/data_types/Conditions.java

```diff
@@ -61,4 +61,14 @@ public String toString() {
             ", operator='" + getOperator() + "'" +
             "}";
     }
+
+    public static boolean areEqual(Conditions a, Conditions b) {
```

Comment:
@notshivansh add this method in Conditions class overriding equals method

Reply:
@shivamrawat101192 added the same.